### PR TITLE
feat: add eks-cn and eks example documents

### PR DESCRIPTION
**Pull request by Vacant2333**
Fixes #

## Proposed Changes

* feat: add eks-cn and eks example documents

**Release Note**

```release-note

```

### CODE REVIEW

File: clusters/ack-ondemand/README.md [hunk 1]
```diff
@@ -1,48 +1,46 @@
-# Demostration Cluster for CloudPilot AI
-
+# Demonstration Cluster for CloudPilot AI
 
 Using this tutorial, you can create a demonstration Aliyun cluster to try CloudPilot AI.
 
 ### Prerequisites
 
 - [aliyun cli](https://help.aliyun.com/zh/cli/?spm=a2c4g.11186623.0.0.18ec478dVSmS6M)
-
```
Comment (jwcesign) [inline clusters/ack-ondemand/README.md]: I think we should keep this

File: clusters/eks/main.tf [hunk 2]
```diff
@@ -67,29 +67,52 @@ module "eks" {
   control_plane_subnet_ids = module.vpc.intra_subnets
 
   eks_managed_node_groups = {
-    # uncomment this block to enable on-demand node group if you want to test the rebalance with on-demand nodes
     default_od_node_group = {
-        name = "od-managed-node-group"
+      name = "od-node-group"
```
Comment (jwcesign) [inline clusters/eks/main.tf]: Let's create two dir called, eks-ondemand, eks-spot

File: clusters/ack-ondemand/README.md [hunk 3]
```diff
@@ -1,48 +1,54 @@
...
 - [aliyun cli](https://help.aliyun.com/zh/cli/?spm=a2c4g.11186623.0.0.18ec478dVSmS6M)
-  
-  If you use Linux, run the following command to install the aliyun cli:
-  ```bash
-  wget https://aliyuncli.alicdn.com/aliyun-cli-linux-latest-amd64.tgz
-  tar -zxvf aliyun-cli-linux-latest-amd64.tgz
-  mv aliyun /usr/local/bin/
-  ```
+
+If you use Linux, run the following command to install the aliyun cli:
```
Comment (jwcesign) [inline clusters/ack-ondemand/README.md]: align with upper content

File: clusters/eks-cn/main.tf [hunk 4]
```diff
@@ -0,0 +1,308 @@
+provider "aws" {
+  region = local.region
+}
+
+data "aws_caller_identity" "current" {}
+data "aws_availability_zones" "available" {}
+
+locals {
+  name            = "cluster-demonstration"
+  cluster_version = "1.29"
```
Comment (jwcesign) [inline clusters/eks-cn/main.tf]: 1.31

File: clusters/eks-cn/README.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+Please note, if you have workloads deployed in your cluster,
+you need to remove these workloads first. Especially, services associated with load balancers (LB)
+may cause dependencies that are cumbersome to resolve manually if not addressed before cluster termination.
+
+```bash
+terraform destroy --auto-approve
+```
+
+## Related
```
Comment (jwcesign) [inline clusters/eks-cn/README.md]: `More`

**Review by jwcesign — COMMENTED** [review]
Other lgtm

File: clusters/ack-ondemand-terway/README.md [hunk 6]
```diff
@@ -1,59 +1,72 @@
...
 export TF_VAR_CLUSTER_NAME=<your_cluster_name>
 terraform init
 terraform apply --auto-approve
 ```
 
 After finish the process, run the following command to get the `kubeconfig`:
+
 ```bash
-export CLUSTER_NAME=<your_cluster_name>
+export CLUSTER_NAME=rentian1
```
Comment (jwcesign) [inline clusters/ack-ondemand-terway/README.md]: Keep `<your_cluster_name>`

File: clusters/eks/main.tf [hunk 7]
```diff
@@ -6,8 +6,8 @@ data "aws_caller_identity" "current" {}
 data "aws_availability_zones" "available" {}
 
 locals {
-  name            = "cluster-demostration"
-  cluster_version = "1.30"
+  name            = "cluster-demonstration"
```
Comment (jwcesign) [inline clusters/eks/main.tf:9]: Let's add a dir called eks-spot

File: clusters/ack-ondemand-terway/README.md (hunk 6 above)
Comment (Vacant2333) [reply clusters/ack-ondemand-terway/README.md]: sorry for this one :(

File: clusters/eks/main.tf [hunk 8]
```diff
@@ -67,29 +67,52 @@ module "eks" {
   control_plane_subnet_ids = module.vpc.intra_subnets
 
   eks_managed_node_groups = {
-    # uncomment this block to enable on-demand node group if you want to test the rebalance with on-demand nodes
-    default_od_node_group = {
-        name = "od-managed-node-group"
+    eks_ondemand_node_group = {
```
Comment (jwcesign) [inline clusters/eks/main.tf]: Let's call this dir `eks-ondemand`

File: clusters/eks/main.tf [hunk 9]
```diff
@@ -67,29 +67,52 @@ module "eks" {
...
-        iam_role_additional_policies = {
-          AmazonEBSCSIDriverPolicy = "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy"
-        }
+      # Needed by the aws-ebs-csi-driver
+      iam_role_additional_policies = {
+        AmazonEBSCSIDriverPolicy = "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy"
+      }
+    }
+
+    eks_spot_node_group = {
```
Comment (jwcesign) [inline clusters/eks/main.tf]: Delete the unrelated code

File: clusters/eks-spot/main.tf [hunk 10]
```diff
@@ -0,0 +1,308 @@
...
+  subnet_ids               = module.vpc.private_subnets
+  control_plane_subnet_ids = module.vpc.intra_subnets
+
+  eks_managed_node_groups = {
+    # eks_ondemand_node_group = {
+    #   name = "eks-ondemand"
+    #
+    #   # amd64
+    #   instance_types = ["t3.xlarge"]
+    #   ami_type       = "AL2_x86_64"
```
Comment (jwcesign) [inline clusters/eks-spot/main.tf]: Delete the unrelated code

**Review by jwcesign — COMMENTED** [review]
Other lgtm

**Review by jwcesign — APPROVED** [review]
/lgtm
